jobs/config.json

```diff
@@ -1200,6 +1200,25 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-e2e-gce-coredns-performance": {
+    "args": [
+      "--cluster=gce-coredns-performance",
+      "--env=CLUSTER_DNS_CORE_DNS=true",
+      "--extract=ci/latest",
+      "--gcp-master-image=gci",
+      "--gcp-node-image=gci",
+      "--gcp-nodes=3",
+      "--gcp-zone=us-east1-b",
+      "--provider=gce",
+      "--test_args=--ginkgo.focus=\\[Feature:PerformanceDNS\\] --gather-resource-usage=master --gather-metrics-at-teardown=master",
```

Comment:
You can keep the --gather* flags, but they probably aren't too relevant for your purpose.